CentralityiOS/CentralityiOS/ViewControllers/ModifyTaskModalViewController.m

```diff
@@ -24,9 +30,63 @@ - (void)viewDidLoad {
     else if([self.modifyMode isEqualToString:kAddTaskMode]){
         [self initModalForAddTaskMode];
     }
+    
+    NSMutableAttributedString *toInput = [[NSMutableAttributedString alloc] initWithString:self.taskTitleInput.text attributes:@{NSForegroundColorAttributeName: [UIColor whiteColor]}];
+    
+    self.taskTitleInput.attributedText = toInput;
+    
+    [self.taskTitleInput addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    self.taskTitleInput.delegate = self;
+    self.taskDescInput.delegate = self;
+    IQKeyboardManager.sharedManager.enable = true;
+}
+
+
+- (void)textFieldDidChange :(UITextField *) textField{
+    NSMutableAttributedString *toInput = [[NSMutableAttributedString alloc] initWithString:textField.text attributes:@{NSForegroundColorAttributeName: [UIColor whiteColor]}];
+    
+    textField.attributedText = toInput;
+    
+    NSArray *todayKeywords = @[@"today", @"now", @"EOD"];
```

Comment:
these seem like they should be constants or member variables! (i could see member variables if you wanted to at some point bring in keywords like this from an external source like a JSON file)

Reply:
I ended up using instance methods to accomplish this